docs/configure/user-interface.md

```diff
@@ -10,4 +10,5 @@ Read more about the Storybook's features below:
 - [Theming](./theming.md)
 - [Storybook addons](./storybook-addons.md)
 - [Sidebar & URLS](./sidebar-and-urls.md)
+- [Toolbar](./toolbar.md)
```

Comment:
Please make this update to all pages under the Configure section.

Reply:
I'm not sure I get this. It seems to me the only places where links are added are here and in the toc?

Reply:
Oh, whoops. Two confusing things happened here:

1. This page ,`user-interface.md`, isn't actually in the TOC and thus isn't built as part of the site.
2. We have a pattern of including all of a sub-section's pages in a list at the bottom of each page in that sub-section ([example](https://storybook.js.org/docs/react/writing-tests/test-runner#learn-about-other-ui-tests)). Based on your change, I thought we were doing that here, but we are not.

Can you just delete this file, please? It shouldn't exist anymore.

Reply:
Sure, I've added a commit deleting it.